test/aura/AuraAvatarTwoTokenTest.t.sol

```diff
@@ -904,20 +882,53 @@ contract AuraAvatarTwoTokenTest is Test, AuraConstants {
         skip(1 weeks);
         vm.expectRevert(
             abi.encodeWithSelector(
-                AuraAvatarOracleUtils.StalePriceFeed.selector, block.timestamp, BAL_USD_FEED.latestTimestamp(), 24 hours
+                AuraAvatarUtils.StalePriceFeed.selector, block.timestamp, BAL_USD_FEED.latestTimestamp(), 24 hours
             )
         );
         vm.prank(keeper);
-        avatar.performUpkeep(new bytes(0));
+        avatar.performUpkeep(abi.encode(uint256(0)));
     }
 
     ////////////////////////////////////////////////////////////////////////////
     // MISC
     ////////////////////////////////////////////////////////////////////////////
 
+    function test_getBptPriceInBal() public {
+        uint256 clPrice = getBptPriceInBal();
+        // Get price from TWAP
+        uint256 twapPrice = fetchBptPriceFromBalancerTwap(IPriceOracle(address(BPT_80BAL_20WETH)), 1 hours);
+
+        // CL price is within 1% of TWAP
+        assertApproxEqRel(clPrice, twapPrice, 0.01e18);
+    }
+
+    function test_getAuraPriceInUsdSpot() public {
+        vm.startPrank(owner);
+        avatar.deposit(10e18, 20e18);
+
+        skipAndForwardFeeds(1 hours);
+
+        (, uint256 pendingAura) = avatar.pendingRewards();
+
+        uint256 spotPrice = getAuraPriceInUsdSpot(pendingAura) / 1e2;
+        uint256 twapPrice = getAuraAmountInUsdc(1e18, 1 hours);
+
+        // Spot price is within 2.5% of TWAP
+        assertApproxEqRel(spotPrice, twapPrice, 0.025e18);
```

Comment:
Why 2.5%?

Reply:
It's arbitrary. The intention was to check that the prices were approximately the same. But since the TWAP lags behind the spot price, they'll never be exactly the same so I just settled at the smallest value that made the test pass

Reply:
SG